packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1796,8 +1796,20 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
       ),
       cause,
     );
+
+    /// The only known current platform that does not bring into view is iOS.
     if (cause == SelectionChangedCause.toolbar) {
```

Comment:
does <kbd>Ctrl-A</kbd> scroll to the end on iOS?

Reply:
This is a good call - I wasn't thinking about the keyboard on iOS world. I'd need someone else to test and verify, I Don't have access to an iPad w/ keyboard.

Good call, I'll ask around tomorrow.

Reply:
Tried on simulator with "Send Keyboard Input to Device" on, it didn't scroll. On macOS it doesn't seem to scroll either (for single/multiline text fields. Only chrome scrolls to the end of the text in its omnibox).